common/remote/sockfile.cpp

```diff
@@ -1964,7 +1967,21 @@ class CRemoteFile : public CRemoteBase, implements IFile
             Owned<CEndpointCS> crit = dirCSTable->getCrit(ep); // NB dirCSTable doesn't own, last reference will remove from table
             CriticalBlock block(*crit);
             sendBuffer.append((RemoteFileCommandType)RFCgetdir).append(dir).append(tail).append(includedirs).append(sub);
-            sendRemoteCommand(sendBuffer, replyBuffer);
+            try
+            {
+                sendRemoteCommand(sendBuffer, replyBuffer);
+            }
+            catch (IDAFS_Exception * e)
+            {
+                //TODO This magic number 8209 should clarify and use an RFSERR_xxx macro instead
```

Comment:
trivial: Probably better to reference that it's the Solaris daliservix.cpp's definition of RFSERR_GetDirFailed and it is unknown why constant is different.